src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelReferenceElementsJsonVisitor.Async.cs

```diff
@@ -0,0 +1,143 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmModelReferenceElementsJsonVisitor.Async.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+using Microsoft.OData.Edm.Vocabularies;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Microsoft.OData.Edm.Csdl.Serialization
+{
+    /// <summary>
+    /// The visitor for outputting Edm reference elements for referenced model.
+    /// </summary>
+    internal partial class EdmModelReferenceElementsJsonVisitor

```

Comment:
Across this file: Why are we calling the sync `IJsonWriter` methods instead of the async equivalents?

Reply:
Same here, `Utf8JsonWriter` does not contain a definition for `Async` methods apart from `FlushAsync` and `DisposeAsync`
`jsonWriter` here is an instance of `Utf8JsonWriter`

[utf8jsonwriter](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter?view=net-8.0)